### DISCUSSION TITLE
Refine border issue

### DISCUSSION THREAD

**Discussion by ihuanh** [cnr-isti-vclab/TagLab#117] [Q&A]
After the new update, the refine border edit function by selecting the area and press "R" not working anymore. 

Since other edit function like merge(M) and Assign class(A) are still working well, I am sure it may not due to the keyborad language setting.

Is it a recent bug or anything relate to my setting?

Windows 11 (x64)
- Python 3.8.8 
- NVCC : 11.3
- GDAL 3.4.3

**Comment by maxcorsini** [comment]
The key setting is not changed, so, by pressing "R" the refine border
should be applied to the current selection.

What happened exactly after pressing "R" ? During the installation you
received some errors ?

Best,
Massimiliano

**Reply by ihuanh** [reply]
Nothing happen and no reponse after pressing "R", I guess the installation is quite normal this time, it is strange that the "R" key function doesn't work........

Cheers,
Yui

**Comment by ihuanh** [comment]
I find the refine issue can be fixed by the suggestion in April by @maxcorsini : 

Refine border not working in latest CPU unstall
https://github.com/cnr-isti-vclab/TagLab/issues/101

//this is a problem raised with the new version of skimage. It will be fixed in the next release of TagLab.
For the moment, a solution is to replace in the file coraline.py (line 44) multichannel=False with channel_axis=None. //

Though I don't know the issue did not fixed in the May update, but it works after I replace the coding as above.
